package.json

```
@@ -28,7 +28,7 @@
     "cross-spawn": "^6.0.5",
     "elementtree": "0.1.7",
     "endent": "^1.1.1",
-    "fs-extra": "^7.0.0",
+    "fs-extra": "^8.0.0",
```

```suggestion
    "fs-extra": "^8.0.1",
```

Any objections if we apply this change, now that fs-extra@8.0.1 is out?

---

Not necessary. Using the caret will update you to the most recent major version. `^8.0.0` will match any 8.x.x release including 8.0.1

---

@GedasGa that is correct. My idea is that the spec should be updated if possible to be consistent with the other Cordova packages where we should apply the latest fs-extra version spec. Not a blocker on my part, though.